pkg/addon-operator/bootstrap.go

```diff
@@ -72,6 +73,7 @@ func (op *AddonOperator) Assemble(modulesDir string, globalHooksDir string, temp
 	shell_operator.RegisterDebugConfigRoutes(debugServer, runtimeConfig)
 	op.RegisterDebugGlobalRoutes(debugServer)
 	op.RegisterDebugModuleRoutes(debugServer)
+	op.OnFirstConvergeDone()
```

Comment:
Why is this required? Could you please leave a comment?

Reply:
Add comment